internal/store/store.go

```diff
@@ -273,11 +273,31 @@ func (bs *BlockStore) LoadBlockCommit(height int64) *types.Commit {
 	}
 	commit, err := types.CommitFromProto(pbc)
 	if err != nil {
-		panic(fmt.Errorf("error reading block commit: %w", err))
+		panic(fmt.Errorf("error from proto block commit: %w", err))
 	}
 	return commit
 }
 
+func (bs *BlockStore) LoadBlockExtCommit(height int64) *types.ExtendedCommit {
+	var pbec = new(tmproto.ExtendedCommit)
+	bz, err := bs.db.Get(extCommitKey(height))
+	if err != nil {
+		panic(err)
+	}
+	if len(bz) == 0 {
+		return nil
+	}
+	err = proto.Unmarshal(bz, pbec)
+	if err != nil {
+		panic(fmt.Errorf("error reading block extended commit: %w", err))
```

Comment:
For info, I did small edits to some of these messages since some of them were identical. Bad idea is you are using a logfile to troubleshoot and you want to be sure _which_ line was hit.
So I am OK with you suggestions to change the wording as long as it doesn't result again in two logs having identical log messages.

Reply:
In a panic, the resulting trace will give the precise location.

Reply:
Good point :blush: